identity-iota/src/did/doc/iota_verification_method.rs

```diff
@@ -90,12 +81,8 @@ impl IotaVerificationMethod {
   ///
   /// If the `fragment` resolves to `Option::None` then the default verification method tag will be
   /// used ("key").
```

Comment:
```suggestion
  /// If the `fragment` resolves to `Option::None` then the default verification method fragment
  /// [`IotaDocument::DEFAULT_METHOD_FRAGMENT`] will be used.
```

Reply:
Thanks for the catch! I made the fragment parameters required when constructing verification methods so there is no default for them now, will update the comments accordingly.